name = "Should Chain Exception";
description = "When catching an exception and rethrowing another one, it is recommended to chain the exception : this will provide both exception to the next catch, allowing better handling with more information.";
